delta/commandsEvents/children/ReplaceChild/ReplaceChild.adoc

```diff
@@ -21,6 +18,9 @@ include::ReplaceChild-diagram.adoc[]
 [horizontal]
 .Parameters
 [[ReplaceChild.newChild, `newChild`]]newChild:: <<chunkType>>
+Subtree to form a new child.
+The _anchor node_ MUST contain the <<ReplaceChild.parent>> as _parent_.{fn-org357}
+All nodes in that subtree MUST either be <<new-nodes>> or <<replaced-nodes>>.
```

Comment:
Why also << replaced-nodes >>? Not even sure what << replaced-nodes >> means in this context.

Reply:
My understanding: the spec used to state that all nodes must be new, now it says that they don't have to be new, but can also be replaced-nodes. This means that any node below `ReplaceChild.replacedChild` can be used as well.  

I cannot remember we discussed this (maybe I wasn't there), but why make this change?